onnxruntime/test/python/transformers/test_optimizer.py

```diff
@@ -132,19 +132,31 @@ def _test_optimizer_on_huggingface_model(
                 self.assertTrue(False)
 
     def test_gpt2_past(self):
-        input = _get_test_model_path("gpt2_past")
-        model = optimize_model(input, "gpt2", num_heads=2, hidden_size=4)
+        for enable_skip_layer_norm_fusion in [False, True]:
+            input = _get_test_model_path("gpt2_past")
+
+            options = FusionOptions("gpt2")
+            options.enable_skip_layer_norm = enable_skip_layer_norm_fusion
+
+            model = optimize_model(
+                input,
+                "gpt2",
+                num_heads=2,
+                hidden_size=4,
+                optimization_options=options,
+            )
 
-        expected_node_count = {
-            "EmbedLayerNormalization": 0,
-            "Attention": 12,
-            "Gelu": 0,
-            "FastGelu": 12,
-            "BiasGelu": 0,
-            "LayerNormalization": 25,
-            "SkipLayerNormalization": 0,
-        }
-        self.verify_node_count(model, expected_node_count, "test_gpt2_past")
+            expected_node_count = {
+                "EmbedLayerNormalization": 0,
+                "Attention": 12,
+                "Gelu": 0,
+                "FastGelu": 12,
+                "BiasGelu": 0,
+                # First LayerNorm is never fused to SkipLayerNorm as it doesn't meet the requirements
+                "LayerNormalization": 25 if not enable_skip_layer_norm_fusion else 1,
```

Comment:
keep the enable_skip_layer_norm_fusion as default?

Reply:
It is enabled by default. This tests whether the fusion works with it disabled or enabled. Basically, it checks if the new changes have nopt broken the old fusions if someone decides to keep SLN disabled.